### ISSUE TITLE
Scheduling the next hybrid meeting - 2024

### ISSUE THREAD

**Issue by titzer** [WebAssembly/meetings#1463]
Hello all,

We had a few discussions at the Oct 2023 meeting about scheduling the following CG meeting and I had suggested that we could do this at Carnegie Mellon University in Pittsburgh, PA. Based on offline discussions with some CG members and our internal staff here at CMU, it's looking like June is 100% doable! This time period works well for CMU's logistics (just after the end of classes, many rooms available).

I'm filing this issue for discussion with the wider community to gather thoughts, constraints, and other considerations.

**Comment by dhil** [comment]
June. Aye!

**Comment by eqrion** [comment]
June works well for me. Slight preference to avoid June 7th, but I think we usually do a midweek thing which wouldn't conflict with that.

**Comment by ashleynh** [comment]
Mid to late June please & thanks

**Comment by michaelficarra** [comment]
No hard constraints, but here's some relevant events in June:

* [102nd TC39 meeting](https://github.com/tc39/agendas/blob/main/2024/05.md)
* [Future Frontend 2024](https://futurefrontend.com/)
* [PLDI 2024](https://pldi24.sigplan.org/)
* [JSConf Budapest](https://ti.to/jsconf-bp/jsconf-budapest-2024)

**Comment by titzer** [comment]
Thanks for the responses so far.

I have had many offline conversations, so just pinging a few random people to weigh in here so we can start the CMU logistics rolling:

@dschuff
@conrad-watt
@dtig
@rossberg 
@lukewagner 
@ajklein 
@justinmichaud
@SPY 
@abrown 
@woodsmc
@wingo

**Comment by justinmichaud** [comment]
https://webengineshackfest.org/ Does seem to also be happening around this time

**Comment by titzer** [comment]
Ok, after some consultation internally with our team, these dates work best for CMU logistics:

* June 4 - 6
* June 11 - 13

We discussed internally to meet Tuesday - Thursday to make travel easier for folks (read: don't have to travel on the weekend).

From the conversation in this thread, the 11 - 13 conflicts with TC39. Is this an issue for any of the Wasm CG participants here? If so, then 4 - 6 might work better.

We could also do the third week of June, which would be 18 - 21, but that starts to run up against PLDI, which other people have expressed as a constraint.

What do people think of the above?

**Comment by conrad-watt** [comment]
I think we should avoid clashing with TC39 if possible - we want to encourage some cross-committee attendance especially if threads is discussed.

@ashleynh does June 4-6 conflict with any events on your/Google's calendar, given your preferences above?

As someone who generally has to plan around PLDI, I could personally deal with the turnaround involved with an 18-21st meeting if that helps other people meet their constraints, although it wouldn't be my first preference!

**Comment by eqrion** [comment]
> I think we should avoid clashing with TC39 if possible - we want to encourage some cross-committee attendance especially if threads is discussed.
> 
> @ashleynh does June 4-6 conflict with any events on your/Google's calendar, given your preferences above?
> 
> As someone who generally has to plan around PLDI, I could personally deal with the turnaround involved with an 18-21st meeting if that helps other people meet their constraints, although it wouldn't be my first preference!

My preference for not doing June 4-6 was that I am going camping on June 7-9. But I can make the turnaround work as travel should be pretty easy for me. Otherwise all the other times look like they work as well.

**Comment by dschuff** [comment]
I would prefer June 11-13 (also due to a weekend event starting Friday the 7th), but I could make 4-6 work too.

**Comment by ashleynh** [comment]
I would prefer June 11 - 13 or June 18 - 21.

**Comment by michaelficarra** [comment]
WasmCon is also happening June 24-25: https://sessionize.com/wasmcon-2024

**Comment by titzer** [comment]
Hello,

Following up on the discussion from last week, I checked internally with our logistics for July/August, and we basically would have these choices:

* July 16-18
* July 30-Aug 1
* Aug 13-15

Of these, the last, Aug 13-15, would be the one with the best spaces available at CMU. 

So the choice is either June 4 - 6 or one of the above dates. I'll follow up with @ashleynh as she has a conflict for the June date, to see if we can work out something (she is the champion of at least one proposal, so it would be good to have an update on it if possible).

Is there a general sentiment to do June, or are any of the above dates significantly preferable?

**Comment by conrad-watt** [comment]
I wouldn't be able to make Aug 13-15. I could make the other dates if necessary, although I'd have a preference for June 4-6.

**Comment by yuri91** [comment]
July 30-Aug 1 does not work for me. Fine for all the other options.

**Comment by michaelficarra** [comment]
July 30-Aug 1 conflicts with a TC39 meeting.

**Comment by ashleynh** [comment]
All of the dates in July & August work for me.

**Comment by rossberg** [comment]
June 4-6 works for me, July 16-18 probably doesn't (and the others seem out already).

**Comment by titzer** [comment]
Thanks everyone for weighing in. I followed up with Ashley offline and we're OK to move forward with June 4-6.

We'll be working to get the signup process and logistics posted soon.

Looking forward to seeing everyone in Pittsburgh then!

We're planning on co-locating research day with the CG meeting as we have the past two times. Tentatively we're thinking that the 6th makes more sense for research day, so the main CG activites will be June 4-5.

**Comment by titzer** [comment]
@conrad-watt has some travel constraints and probably cannot make June 4th. As a CG chair it's important for him to be there for the CG days, so he's asked if Research Day could be June 4 and June 5-6 for the CG days. I know @eqrion has a constraint on the 6th.

Does anyone have other constraints or preferences on the ordering of days within the June 4-6 window?

**Comment by conrad-watt** [comment]
Thanks @titzer for considering this, and I'm sorry that my new constraint has come up at the last minute after I initially said the June 4-6 dates would work well for me.

**Comment by michaelficarra** [comment]
@titzer Do we have more precise location information yet, such as building/room numbers? And times?

**Comment by titzer** [comment]
Yep, it will be in [Gates & Hillman Centers](https://maps.app.goo.gl/yG2fozWrGsf4GMua8) 6115. I'll add that to the agenda.

**Comment by dtig** [comment]
There's a [linked doc](https://docs.google.com/document/d/1Hjfl6QnqVQuXjFZ9vuP0vTMl0sQhBhupu_vfg2XbE5Q/edit?usp=sharing) in the logistics section that has the exact address, recommendations for near by hotels and other useful information!

**Comment by sunfishcode** [comment]
The 2024 in-person meeting [happened](https://github.com/WebAssembly/meetings/blob/main/main/2024/CG-06.md)!
